### PR TITLE
feat: Provide owner on constructor

### LINKED ISSUE #50: Set owner on constructor
This prevents the need of sending an extra transaction to change it later.

### PR DESCRIPTION
**Pull request by fzavalia**
Closes #50

### CODE REVIEW

File: test/SubdomainWithDCLControllerV2.spec.js [hunk 1]
```diff
@@ -1957,11 +1963,27 @@ describe('DCL Names V2 with DCLControllerV2', function () {
...
+        const owner = await contract.owner()
+        expect(owner).to.be.equal(controllerOwner)
+      })
+
+      it('should accept owner the same as deployer', async function () {
+        const contract = await DCLControllerV2.new(
+          manaContract.address,
+          dclRegistrarContract.address,
+          feeCollector,
+          deployer,
```
[the remark this answers is not in the document]
Comment (fzavalia) [reply test/SubdomainWithDCLControllerV2.spec.js:1981]: The previous test contemplates this

[the remark this answers is not in the document]
Comment (fzavalia) [reply test/SubdomainWithDCLControllerV2.spec.js:1981]: https://github.com/decentraland/avatars-contract/pull/51/files#diff-be114b8ef561a5fe95d81ecb95273d93684fa9a50b0221cd4d28c29bfa54cdaaR1973 this one

File: contracts/ens/DCLControllerV2.sol [hunk 2]
```diff
@@ -33,7 +33,7 @@ contract DCLControllerV2 is Ownable {
      * @param _registrar - address of the DCL registrar contract
      * @param _feeCollector - address of the fee collector
 	 */
```
Comment (nachomazzara) [inline contracts/ens/DCLControllerV2.sol:35]: Can we add the dev notation for the new param?

Comment (fzavalia) [reply contracts/ens/DCLControllerV2.sol:35]: Completely forgot, nice catch

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**